### PR TITLE
docs: contributing updates

### PR DESCRIPTION
**Pull request by chri2547**
**What this PR does / why we need it**: Consolidates and refreshes docs contributing guidelines.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: contribute/documentation/README.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+- Train the dog
+
+**References**
+
+- List of dog equipment you will need
+- Table of breeds that includes breed name, size range, short or long hair, and type of dog
+
+#### Prepare your environment
+
+Before you begin writing, we recommend that you fork and clone the Grafana repository so that you can use a text editor locally to create branches, commit your changes, and create a PR.
```
Comment (GrafanaWriter) [inline contribute/documentation/README.md:130]: fork and clone, or do we just want folks to clone?

File: contribute/documentation/README.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+When a PR is added to the repo with a label associated with `docs`, it will be reviewed by a member of the technical writing team. Depending on the size of the PR and the priority of other work, the PR will either be immediately reviewed and merged (minor fixes typically follow this pattern) or the PR will be triaged and placed in the backlog of work or moved into further development.
+
+## Join our community
+
+For general discussions on documentation, you’re welcome to join the `#docs` channel on our [public Grafana Slack](http://slack.raintank.io) team.
+
+## Reference: Top five writing tips
+
+While we can't provide a comprehensive style guide to this document, we thought providing a few writing tips to help the writing process.
```
Comment (GrafanaWriter) [inline contribute/documentation/README.md:192]:
```suggestion
While we don't provide a comprehensive style guide in this document, here's a few writing tips to help the writing process.
```

**Review by GrafanaWriter — APPROVED** [review]
Thank you!

**Review by osg-grafana — CHANGES_REQUESTED** [review]
Requires syncing with release team wrt timing. Blocking for now.

**Review by osg-grafana — APPROVED** [review]

File: contribute/documentation/README.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+## Understanding the structure of Grafana documentation
+
+All Grafana Enterprise and OSS documentation is located in the [Grafana open source project](https://github.com/grafana/grafana) GitHub repository: https://github.com/grafana/grafana/tree/main/docs/sources.
+
+- The **sources** directory organizes content by topic areas, for example **administration** and **alerting**.
+- Topic directories include an `_index.md` file, which provides an overview of the topic, and optionally includes subtopics that provide more detail.
+
+> The `_index.md` file is required.
+
+### Writing in markdown
```
Comment (osg-grafana) [inline contribute/documentation/README.md:22]:
```suggestion
### Writing in Markdown
```

Comment (osg-grafana) [reply contribute/documentation/README.md:22]: Change markdown to Markdown throughout.

File: contribute/documentation/README.md (hunk 1 above)
Comment (osg-grafana) [reply contribute/documentation/README.md:130]: @achatterjee-grafana, do folks need to fork from Grafana to make a change rather than just clone?

**Review by osg-grafana — APPROVED** [review]

File: contribute/documentation/README.md (hunk 1 above)
Comment (achatterjee-grafana) [reply contribute/documentation/README.md:130]: External users will fork, not clone the repo.

**Pull request merged**